BuildScripts/lib/GetCommonList.ps1

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env pwsh
+#Requires -Version 7.2
+
+[CmdletBinding()]
+param (
+  [Parameter(Mandatory)]
+  [string]
+  $Path
+)
+
+New-Item -Path (Join-Path $PSScriptRoot "CommonLibList.txt") -ItemType File -Force
+
+Get-ChildItem (Join-Path $Path "Windows") | ForEach-Object {
+  $Private:DXFile = (Join-Path $Path "Windows" $($_.Name))
+  $Private:GLFile = (Join-Path $Path "OpenGL" $($_.Name))
+  $Private:XNAFile = (Join-Path $Path "XNA" $($_.Name))
```

Comment:
```powershell
Build-Project -Configuration $Configuration -Game Ares -Engine WindowsDX -Tar...
```
part of [Rans4ckeR/xna-cncnet-client/net6/BuildScripts/lib/BuildTools.ps1#L37-L43](https://github.com/Rans4ckeR/xna-cncnet-client/blob/net6/BuildScripts/lib/BuildTools.ps1#L37-L43)

In PowerShell, `Ares` and `WindowsDX` here are not an enum, but string.
PowerShell will automatically try to convert the string into an enum.

Same thing here, changing `"Windows"` to `Windows` is meaningless because both are string values.

The correct way to call an enum is `[Type]::Value`.
For example `[Engines]::WindowsDX`.

But it's a pain, isn't it?

Reply:
> But it's a pain, isn't it?

Why did you introduce it in the other places then?

Reply:
Powershell will throw an exception when the string convert to enum failed.